scripts/docker-compose.yml

```diff
@@ -9,12 +17,12 @@ services:
     volumes:
       - ${MONTAGU_ORDERLY_PATH}:/orderly
   api:
-    image: docker.montagu.dide.ic.ac.uk:5000/montagu-api:${MONTAGU_API_VERSION}
+    image: docker.montagu.dide.ic.ac.uk:5000/montagu-api:master
```

Comment:
I just removed these extra bits of config because these scripts are complicated enough, and I figured we only want to track master anyway.